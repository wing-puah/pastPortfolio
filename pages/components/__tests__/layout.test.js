import { shallow, mount } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Layout, { BodyContainer, BodyContainerWithCss } from '../navbar';

describe( '<Layout/>', () => {
  it('matches the snapshot', () => {
    const wrapper = shallow(<Layout />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  })
})
