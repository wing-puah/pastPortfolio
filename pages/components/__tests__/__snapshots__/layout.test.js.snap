// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout/> matches the snapshot 1`] = `
<Navbar
  classes={
    Object {
      "navLink": "Navbar-navLink-1",
    }
  }
/>
`;
